worksheet_D/TicTacToe.py

```diff
@@ -0,0 +1,114 @@
+class OxoBoard:
+
+    def __init__(self,x,y):

```

Comment:
Why are `x` and `y` needed here? You do not seem to use `self.x` and `self.y` anywhere